### ISSUE TITLE
Weird handling of numbers in lists

### ISSUE THREAD

**Issue by sedric** [Rolisteam/DiceParser#153]
Hi,

To make a macro that implement Call of Cthulhu V7 rolls, on which bonuses/maluses are in the form of X dices with tens faces and keep the lowest / highest, I need to implement a D100 that fully emulate the D tens + D10 mechanism.

I tried to do it with list but it seems numbers are treated weirdly : you can do arithmetic with multiple pool dices but you can't use operators and the output is somewhat broken.

If I want to list words (here: `2L[sword,bow,knife,gun,shotgun]`), it's treated like this : 
```
shotgun,sword
```

If I want to list numbers, on the other hand, it's treated like a normal dice roll but as you can see, it doesn't show the dice output : 
```
# 30
Details:[1L[0,10,20,30,40,50,60,70,80,90] ()]
```

And when you do multiple roll it doesn't show both values but add the values as result : 
```
# 100
Details:[2L[0,10,20,30,40,50,60,70,80,90] (0 0)]
```

So when you try to use other operators, it's just more confusing as they are not applied and this is visible only on high values : 
```
# 130
Details:[2L[0,10,20,30,40,50,60,70,80,90]k (0 0)]
```

But, still, adding other dices values work :
```
# 139
Details:[2L[0,10,20,30,40,50,60,70,80,90]k+1D10 (0 0 9)]
```

Is it possible to have a more consistent behavior ? Either treat all output as strings / numbers as "numbered roll" / throw an error, but this middle ground is confusing and undocumented.

**Comment by obiwankennedy** [comment]
Yes this behaviour is strange but you can do the same in another way.

dice "D[0..9]*10;D[0..9]*10;1d10;[\$1,\$2]k1+\$3;\"result: \$4 [\$1,\$2,\$3]\""
